src/ray/core_worker/core_worker.h

```diff
@@ -1384,6 +1384,9 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
   std::shared_ptr<rpc::RuntimeEnvInfo> OverrideTaskOrActorRuntimeEnvInfo(
       const std::string &serialized_runtime_env_info) const;
 
+  std::shared_ptr<rpc::RuntimeEnvInfo> OverrideTaskOrActorRuntimeEnvInfoImpl(
+      const std::string &serialized_runtime_env_info) const;
+
```

Comment:
pls add a comment here explaining why we have the separation btw `OverrideTaskOrActorRuntimeEnvInfo` and `OverrideTaskOrActorRuntimeEnvInfoImpl`

Reply:
sure